src/cli/traceViewer/web/ui/timeline.css

```
@@ -72,19 +73,26 @@
   bottom: 0;
   background-color: red;
   border-radius: 3px;
+  --action-color: 'transparent';
+  background-color: var(--action-color);
+}
+
+.timeline-action.selected {
+  filter: brightness(70%);
+  box-shadow: 0 0 0 1px var(--action-color);
```

This renders differently from an outline??

```suggestion
  outline: 1px solid var(--action-color);
```

---

Yep, it respects border-radius!